brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post Layout - Conflicting with the Elvira gallery plugin.

### Description

- https://trello.com/c/McCgRu58/300-fix-post-excerpt-conflicting-with-the-elvira-gallery-plugin

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install NextGen Gallery and Elvira Gallery plugin.
- Switch to `post-excerpt-issue` and update the page. It updates successfully.
- Add an excerpt to the post from meta settings and then check the post layout at the front end. (It should display an excerpt in this case)

### Checklist:
